src/main/java/org/aksw/simba/squirrel/sink/impl/sparql/QueryGenerator.java

```diff
@@ -86,16 +93,23 @@ public Query getSelectQuery(CrawleableUri uri, Triple triple) {
      * @return
      */
     public Query getSelectQuery(CrawleableUri uri, Triple triple, boolean bSelectAll) {
-        String strQuery = "SELECT ?subject ?predicate ?object WHERE { GRAPH <" + uri.getUri().toString() + "> { ";
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("SELECT ?subject ?predicate ?object WHERE { GRAPH <");
+        stringBuilder.append(uri.getUri());
+        stringBuilder.append("> { ");
         if (bSelectAll) {
-            strQuery += "?subject ?predicate ?object ";
+            stringBuilder.append("?subject ?predicate ?object ");
         } else {
-            strQuery += triple.getSubject().getName() + " " + triple.getPredicate().getName() + " " + triple.getObject().getName() + " ; ";
+            stringBuilder.append(triple.getSubject().getName());
+            stringBuilder.append(" ");
+            stringBuilder.append(triple.getPredicate().getName());
+            stringBuilder.append(" ");
+            stringBuilder.append(triple.getObject().getName());
```

Comment:
This won't work. Please see the javadoc of [`Node::getName`](https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/graph/Node.html#getName--). Especially the second part of the sentence: "...otherwise die horribly." :laughing: 

For creating an `INSERT` query, you may want to reuse the following piece of code: https://github.com/hobbit-project/core/blob/master/src/main/java/org/hobbit/storage/queries/SparqlQueries.java#L719-L743
Unfortunately, I am not aware of a similar way to easily create `SELECT` queries.